packages/webapp/pages/squads/[handle]/index.tsx

```diff
@@ -71,6 +70,14 @@ const SquadEmptyScreen = dynamic(
     ),
 );
 
+const SquadLoading = dynamic(
+  () =>
+    import(
+      /* webpackChunkName: "squadLoading" */ '@dailydotdev/shared/src/components/errors/SquadLoading'
+    ),
+  { ssr: false },
+);
```

Comment:
There are certain components that we don't need to be loaded from the server side. Especially those components that are dynamic and change on load. One example is this, the layout changes based on the screen size which React throws an error if we don't disable it here.

Reply:
Also weird, it should only be different in loading in client and/or server bundle, would not 100% brush it off before going to prod.

Reply:
Yeah, I was also surprised to see these kinds of issues.

![Screenshot 2024-07-02 at 3 54 56 AM](https://github.com/dailydotdev/apps/assets/13744167/fcda7240-b019-49e3-8f5a-782f3a67c40d)